src/subcommand/wallet/send.rs

```diff
@@ -53,6 +53,17 @@ impl Send {
       },
     };
 
+    for inscribed_satpoint in &inscription_satpoints {
```

Comment:
I think it would be better to move this check into the transaction builder, and also add a unit test for it, so that whenever we use the transaction builder it does the same check. We can also add another error variant to the transaction builder error type, something like `UtxoContainsAdditionalInscription`.

Reply:
I added it to the `select_ordinal()` function. If you want I can also make a seperate builder function for it but then we will have to change all the unit tests.